Testscripts/Linux/TIME-CLOCKSOURCE.sh

```diff
@@ -27,16 +27,16 @@ CheckSource()
 {
 	current_clocksource="/sys/devices/system/clocksource/clocksource0/current_clocksource"
 
-	# Microsoft LIS installed version has lis_hyperv_clocksource_tsc_page
-	# Without Microsoft LIS, hyperv_clocksource_tsc_page
-	# CentOS 6.8 or older versions, hyperv_clocksource
-	clocksource="hyperv_clocksource_tsc_page"
-	mj=$(echo "$DISTRO_VERSION" | cut -d '.' -f 1)
-	mn=$(echo "$DISTRO_VERSION" | cut -d '.' -f 2)
-	if [[ $DISTRO_NAME == "centos" || $DISTRO_NAME == "rhel" ]] && [[ $mj -lt 7 && $mn -lt 9 ]]; then
+	# Microsoft LIS installed version has lis_hyperv_clocksource_tsc_page,
```

Comment:
Please use 'or' instead of comma.

Reply:
Fixed in 6e85d8bb0333e43cf3ad087c88b000fcfbbe74c5